00 Core/MWSE/mods/tew/AURA/Interior Weather/interiorWeatherMain.lua

```diff
@@ -15,7 +17,7 @@ local sound
 local interiorTimer, scalarTimer, transitionScalarLast
 local cell, cellLast, interiorType, weather, weatherLast
 local thunRef, thunder, thunderTimer, thunderTime
-local thunArray = common.thunArray
+local thunArray = config.thunderSounds and soundData.thunders or common.thunArray
```

Comment:
Ha! I was about to suggest we do this - many thanks 👐

Reply:
Can keep lead time for replacers in this case. If people don't want new thunder sounds, they'll hear replacer thunder with lead time, both interior and exterior.